dll/win32/framedyn/WbemTime.cpp

```diff
@@ -0,0 +1,522 @@
+/*
+* PROJECT:     ReactOS Kernel
+* LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:     Implementation of WBEMTime and WBEMTimeSpan classes.
+* COPYRIGHT:   Copyright 2017 Jin Juanshi (2033537949@qq.com)
+*/
```

Comment:
`ReactOS Kernel` is actually an _exempli gratia_ as shown in the Wiki, this must be changed that fits accordingly your purpose (on this case that would be `ReactOS system libraries`).

Reply:
I've fixed.